functions/_fk_clone.fish

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env fish
+
+function _fk_clone -a repository
+    if echo $repository | grep '/' > /dev/null
+        _fk_github_clone $repository $repository
+    else
+        _fk_github_clone "$fk_github_user/$repository" $repository
+    end
+
+    if test $status -eq 0
```

Comment:
Fish does not support "strict mode" like bash (yet) https://github.com/fish-shell/fish-shell/issues/510